content/authorization/application-registration-prerequisites.md

```diff
@@ -124,25 +122,39 @@ Please follow one of below approaches to check TLS Certificates for your applica
        * [https://www.ssllabs.com/ssltest/index.html](https://www.ssllabs.com/ssltest/index.html)
        * [https://ssltools.digicert.com/checker/](https://ssltools.digicert.com/checker/)
        * [https://www.thesslstore.com/ssltools/ssl-checker.php](https://www.thesslstore.com/ssltools/ssl-checker.php)
+   1. Using the Firefox web browser to check the certificate.
+       * Load the URL into the browser
+       * Click on the Lock symbol in the browser's address bar
+       * Click "Connection Secure" 
+       * Click "More Information" 
+       * Click "View Certificate"
+       
+
 
 What you should pay attention to:
 
-   1. Supported TLS Certificates types:
+   1. Supported TLS Certificates types (this is usually in a section called "Certificate Policies" as is the case if you view the cert in Firefox)  :
        * Domain Validated (DV),
        * Organization Validated (OV),
        * Extended Validation (EV).
-   1. Supported TLS protocols:
+ ##### Example
+ 
+ Using Firefox to view your TLS certificate may result in a similar Organization Validated certificate as in the example below:
+ [Organization Validated Certificate Type](/images/cert_policies.png)
```

Comment:
This formats a bit oddly - a numbered list interrupted by an outdented aside - but I'm not expert enough with Markdown / Nanoc to be sure there's a better way...

Reply:
Honestly, neither am I (an expert) and am not sure of a better route. I am making changes in the web ui which does not show the images in the "Preview" of the document. (I will use the local app next time...)